Makefile

```diff
@@ -67,6 +67,11 @@ build-integration: ## Build integration test binary
 test-e2e: ## Run openshift specific e2e test
 	$(MAKE) -C ./vendor/github.com/openshift/cluster-api-actuator-pkg $@
 
+.PHONY: test-sec
+test-sec:
+	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
```

Comment:
should we run this in containerized env like  `gen-crd` , `update-codegen` etc?

Reply:
I don't think so, since this doesn't really generate code. It merely prints out a report if security issues are found.

Reply:
I think the premise is: if this has value and finds problems then we should gate CI on the result of running the tool. It's not about generating code, more that we have a rule that runs this tool during CI.

Reply:
@frobware right! The first step is to add this so folks can run it through make; the next step is to add it to CI. We already added this for the machine-config-operator: https://github.com/openshift/release/pull/4273